lib/beaker/host/unix/exec.rb

```diff
@@ -138,7 +138,9 @@ def get_ip
     if self['platform'].include?('solaris') || self['platform'].include?('osx')
       execute("ifconfig -a inet| awk '/broadcast/ {print $2}' | cut -d/ -f1 | head -1").strip
     else
-      execute("ip a | awk '/global/{print$2}' | cut -d/ -f1 | #{self['hypervisor'] == 'vagrant' ? 'tail' : 'head'} -1").strip
+
+      pipe_cmd = "#{self['hypervisor']}".include?('vagrant') ? 'tail' : 'head'
```

Comment:
Why is the string interpolation needed? Is it in case it's nil? Isn't it better to write:
`pipe_cmd = self['hypervisor']&.include?('vagrant') ? 'tail' : 'head'`